services/db/db.js

```diff
@@ -6,7 +6,8 @@ const getDbConn = (_ => {
     let dbConn = null;
     return dbAddr => {
         if (!dbConn) {
-            dbConn = pgp({})(process.env.DATABASE_URL) || pgp({})(dbAddr);
+           // dbConn = pgp({})(process.env.DATABASE_URL) || pgp({})(dbAddr);
+           dbConn = pgp({})(dbAddr);
```

Comment:
This you probably just forgot to change back, but just in case; remember that `dbConn = pgp({})(dbAddr);` is only to test locally. If you submit a PR with that `dbConn` and it's accepted, Travis will automatically deploy on Heroku and no DB interaction will ever be achieved since `pgp({})(dbAddr)` is not the proper address to connect to Heroku's DB.

Reply:
Yes, my bad I did forget to do that. I will re submit PR with updates on my route as advised by Aditya and this. thank you!